Basim_Muhammad/registration/server.py

```diff
@@ -0,0 +1,64 @@
+from flask import Flask,render_template,request,redirect,flash,session
+from flask.ext.bcrypt import Bcrypt
+import re
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+NAME_REGEX=re.compile(r'^[A-Za-z]')
+
+from mysqlconnection import MySQLConnector
+app = Flask(__name__)
+bcrypt = Bcrypt(app)
+app.secret_key = 'ThisIsSecret'
+
+mysql = MySQLConnector(app, 'users')
+@app.route('/')
+def index():
+	# all_query = mysql.query_db("SELECT * FROM users")
+	# print all_query
+	return render_template('index.html')
+
+@app.route('/login', methods=['POST'])
+def login_data():
+	if request.form['action']=='registration':
+		first=request.form['first_name']
+		last=request.form['last_name']
+		email=request.form['email']
+		password=request.form['password']
+		confirm=request.form['password_confirmation']
+		if len(first)<2 or len(last)<2 or len(email)<1 or len(password)<8:
```

Comment:
I know it's a lot more writing but it's best to separate these out.  That way you can specifically tell the user which fields the error is in.  For example if you had optional fields mixed with mandatory fields the user should know which ones they're missing